ci/release_notes.md

```diff
@@ -0,0 +1,33 @@
+# Software Updates
+
+- Updated BOSH to 270.11.0 from 270.10.0
+- Updated Azure CPI to 37.1.0 from 37.0.0
+- Updated Postgres job to 10 from 9.4
```

Comment:
move the postgres job change to a section for Component Changes

Reply:
Added that section and moved the update there.